command/agent/host/freebsd.go

```diff
@@ -0,0 +1,51 @@
+// +build freebsd
+
+package host
+
+import (
+	"syscall"
+)
+
+// IntToString converts an integer to a string
+func IntToString(orig []int8) string {
```

Comment:
The docstring and name of this function is strange: it doesn't convert an integer to a string, but an array of short ints to a string. And the return value is an empty string for a bunch of arbitrary inputs, which seem like they should returning an error instead.

There's also probably no reason to make this exportable (upper-cased). This also doesn't compile: the caller below calls an unexported version of this function.

Reply:
Oops, it appears I goofed on this. I swiped IntToString from gopsutil. It appears I need to submit a PR to them before I can continue with this further. I am the port maintainer for nomad on FreeBSD. I generally have to provide fixes to get nomad to build on FreeBSD, so instead of making patches, I thought I would go ahead and just start submitting PR's ;-)

Reply:
@jhixson74 Thanks for the PR.  I see that this file isn't a vendored file - so I would suggest making the changes without necessarily waiting for gopsutil to change terminology.